lib/coinbase/balance.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of an Balance.
```

Comment:
`of a Balance` or `of an Asset Balance`?

Reply:
What's the difference in your opinion there? We can clarify it is an amount of a specific asset?